doc/source/serve/advanced-guides/performance.md

```diff
@@ -75,7 +75,24 @@ Ray Serve allows you to fine-tune the backoff behavior of the request router, wh
 - `RAY_SERVE_ROUTER_RETRY_MAX_BACKOFF_S`: The maximum backoff time (in seconds) between retries. Default is `0.5`.
 
 
-### Give the Serve Controller more time to process requests
+### Enable throughput-optimized flags
+
+:::{note}
+In Ray v2.54.0, the defaults for `RAY_SERVE_RUN_USER_CODE_IN_SEPARATE_THREAD` and `RAY_SERVE_RUN_ROUTER_IN_SEPARATE_LOOP` will change to `0` for improved performance.
+:::
+
+Ray Serve offers performance flags that improve throughput and latency. You can enable all optimizations at once with `RAY_SERVE_THROUGHPUT_OPTIMIZED=1`, or configure individual flags:
```

Comment:
```suggestion
This section details how to enable Ray Serve options focused on improving throughput and reducing latency.

To configure all options to the recommended settings, set the environment variable `RAY_SERVE_THROUGHPUT_OPTIMIZED=1`.

You can also configure each option individually. The following list details the recommended configurations and their impact:
```

Reply:
>can I still override individual configs below?

no, need to unset RAY_SERVE_THROUGHPUT_OPTIMIZED and manually set each one individually.